Sources/MockoloFramework/Models/VariableModel.swift

```diff
@@ -29,7 +41,7 @@ final class VariableModel: Model {
     }
 
     var underlyingName: String {
-        if isStatic || type.defaultVal() == nil {
```

Comment:
The computed property is generated as below, but I think it is meaningless, so I will remove it.
https://github.com/uber/mockolo/pull/267/files#diff-0680dee8c19da68e406e1ecf918c1c39fe5c55214b3e399ac8dd929fe6b72c7cL43-L48